src/shelf.js

```diff
@@ -22,23 +22,23 @@ const formatUseCaseDoc = (usecase) => {
 
   if (usecase.request) {
     if (Object.entries(usecase.request).length == 0)
-      responseParams.push({ 'name': 'Object of', 'type': usecase.request.name })
+      responseParams.push({ name: 'Object of', type: usecase.request.name })
     else {
       for (const [key, value] of Object.entries(usecase.request)) {
-        requestParams.push({ 'name': key, 'type': value.name ? value.name : `Array of ${value[0].name}` })
-
+        requestParams.push({ 
+          name: key, type: value.name ? value.name : { iterableKind: 'Array of', valueOf: value[0].name } 
```

Comment:
i think we could to refactoring it and use a map instead a loops with an array, what do you think about?(optional)

Reply:
I think that it's great.